documentation/src/main/asciidoc/topics/ref_server_interfaces.adoc

```diff
@@ -1,6 +1,8 @@
-[id='server_interfaces-{context}']
-= Server Interfaces
-{brandname} servers can use different strategies for binding to IP addresses.
+[id='network-interfaces_{context}']
+= Network interfaces
+{brandname} Server can use different strategies for binding to IP addresses.
+
+By default, {brandname} Server multiplexes endpoints to a single TCP/IP port and automatically detect protocols of inbound client requests.
 
 == Address Strategy
```

Comment:
what is the difference between this and `Network Address Strategy` a couple of lines below?

Reply:
I'm probably confused about this section xD since the comment above and `Link Local Strategy` and `Site Local Strategy` use the same config: `<inet-address value="x.y.z.w"/>`

Reply:
hmm. yeah I think I need to review this topic again. will take a look in the morning. if it's going to require large changes I'd rather split into a separate issue than expand the scope of the current PR.

Reply:
ok, this should be all sorted now. was a little disjointed and messy but I think I've fixed it.